packages/sdk-v2/src/sdk/claim/react.ts

```diff
@@ -85,14 +94,14 @@ export const useMultiClaim = (poolsDetails: PoolDetails[] | undefined) => {
     const { errorMessage = "", status } = state;
     const isError = isTxReject(errorMessage) || status === "Exception";
 
+    if (claimedContracts.length === poolContracts?.length && !["None", "PendingSignature"].includes(status)) {
```

Comment:
explain the logic change

Reply:
We show a modal 'sign transaction' and a modal 'await transaction to be validated'
isClaiming boolean is used for this.
before we set isClaiming only after success of mining, so all the user saw was sign transaction modal (for gooddapp